commit f4b3d6b7692332c329b567d91460b8ada96741e2
Author: Neha Ojha <nojha@redhat.com>
Date:   Thu Dec 3 21:54:23 2020 +0000

    mgr/ActivePyModules.cc: don't log config value in get_typed_config
    
    Signed-off-by: Neha Ojha <nojha@redhat.com>

diff --git a/src/mgr/ActivePyModules.cc b/src/mgr/ActivePyModules.cc
index 0d69729d59a..a403eebd0b7 100644
--- a/src/mgr/ActivePyModules.cc
+++ b/src/mgr/ActivePyModules.cc
@@ -610,7 +610,10 @@ PyObject *ActivePyModules::get_typed_config(
         derr << "Module '" << module_name << "' is not available" << dendl;
         Py_RETURN_NONE;
     }
-    dout(10) << __func__ << " " << final_key << " found: " << value << dendl;
+    // removing value to hide sensitive data going into mgr logs
+    // leaving this for debugging purposes
+    // dout(10) << __func__ << " " << final_key << " found: " << value << dendl;
+    dout(10) << __func__ << " " << final_key << " found" << dendl;
     return module->get_typed_option_value(key, value);
   }
   PyEval_RestoreThread(tstate);
